Tribler/Test/test_as_server.py

```diff
@@ -465,6 +477,8 @@ def callLater(self, seconds, callback):
                 callback()
 
     def quit(self):
+        # Martijn: stop wx thread watchdog event sooner to prevent errors when yielding in the GUI shutdown
+        self.watchdog.unregister_event("wx thread")
```

Comment:
@whirm I agree but during a `wx.Yield`, the wx watchdog event is not set. This is problematic if the yield takes a long time (> 10 seconds). I've only seen the tripping happening when closing Tribler during the tests but the tests themselves do not fail if it trips. It's also really hard to reproduce this issue with the tests. It only happens in around 5% of the test cases and mostly on the tests that takes a long time (i.e. channel search).

Can you think of a better solution for this problem? Maybe we can ignore the stack trace for now. Nevertheless, I will remove the commit with the code above since this is not really a good solution for now.